tests/api.test.ts

```diff
@@ -217,14 +217,11 @@ describe('api', () => {
         const result = await explore(['data/foo.min.js', 'data/no-map.js']);
 
         expect(result.bundles.length).to.eq(1);
-        expect(result.errors.length).to.eq(1);
+        expect(result.errors.length).to.eq(2);
       });
 
       it('should add error when used with bad sourcemap', async function() {
-        const result = await explore({
-          code: 'data/no-map.js',
-          map: 'data/foo.min.no-map.bad-map.js.map',
-        });
+        const result = await explore('data/foo.min.no-map.bad-map.js');
```

Comment:
To keep things simple and avoid potential future problems, I didn't reuse generated source with different source maps. `foo.min.no-map.bad-map.js` has a corresponding `foo.min.no-map.bad-map.js.map` file.